d3a_interface/schemas.py

```diff
@@ -34,16 +34,16 @@ class ScenarioSchemas:
                     "uuid": {"type": "string"},
                     "libraryUUID": {"anyOf": [{"type": "string"}, {"type": "null"}]},
                     "children": {"anyOf": [{
-                                    "type": "array",
-                                    "items": {"anyOf": [
-                                        {"$ref": "#/definitions/area"},
-                                        {"$ref": "#/definitions/pv"},
-                                        {"$ref": "#/definitions/load"},
-                                        {"$ref": "#/definitions/infinite_power_plant"},
-                                        {"$ref": "#/definitions/finite_power_plant"},
-                                        {"$ref": "#/definitions/storage"}
-                                    ]}, "default": []},
-                                    {"type": "null"}]}
+                        "type": "array",
+                        "items": {"anyOf": [
+                            {"$ref": "#/definitions/area"},
+                            {"$ref": "#/definitions/pv"},
+                            {"$ref": "#/definitions/load"},
+                            {"$ref": "#/definitions/infinite_power_plant"},
+                            {"$ref": "#/definitions/finite_power_plant"},
+                            {"$ref": "#/definitions/storage"}
+                        ]}, "default": []},
+                        {"type": "null"}]}
```

Comment:
I see a lot of whitespace corrections in your PRs. Are you using a special linter that complains? Or are these just personal preference?

Reply:
I am using Pycharm's linter , let me check its settings

Reply:
@hannesdiedrich This is from the Pycharm official docs : 

> PyCharm adheres to PEP8 rules and requirements for arranging and formatting Python code.

I can be extra cautious when refactoring the code in the future

Reply:
Hmm, so pycharm added these whitespace changes for you? 
I am also using the pycharm linter, but is is only notifying me with grey underlines and it is not notifying me in these places here. Which version of pycharm are you using?

Reply:
I am using Pycharm 2019.3.1 , maybe try pressing `Command` + `Shift` + `L` ?

Reply:
In pyCharm:
- `Command` + `Option` + `L`: reformat code
- `Command` + `Option` + `Shift` + `L`: reformat file

I think it's not the default behaviour on save, but rather an action that you can trigger manually.

The issue I see with committing this kind of changes is that they hide the history of that specific line in the git tree. For example, if I want to know why some code was implemented/edited, I would usually use git blame or check the history. However, if the line was changed because of linting fixes, I would find commits that are unrelated to that line. In theory, linting should be done when the line is created (but it's not always the case, especially with older code).

Reply:
Yeah, cant you just omit pressing  Command + Shift + L ?
I agree with Pierpaolo and add, that these changes also mask the real changes in PRs (in this PR it is just one line, right?)

Reply:
Yes , Deal .

Reply:
also agree with @fievelk , just to cover all cases here, @Hassan754 can you also check your Pycharm settings? Mine are like this:
![tabsvsspaces](https://user-images.githubusercontent.com/37290802/104907046-c7593700-5984-11eb-83e4-06f20c9bf301.png)

Reply:
![Screen Shot 2021-01-18 at 13 05 12](https://user-images.githubusercontent.com/38125715/104907502-dee8ed80-598d-11eb-8720-74297bcccb57.png)

Reply:
@spyrostz There is an option to export/import settings , maybe if you find this refactoring style convenient , all of us can agree to use it

Reply:
Good, so your tabs / space settings are intact. No need for exporting the settings since we all use the default Pycharm ones, the only reason why I asked you to check it out was to rule out the case that these settings were modified and were the root cause of the diff (they were not).